train.py

```diff
@@ -94,9 +94,9 @@ def evaluate_model(
 
 			for r in sum(stats, []) if args.verbose else []:
 				print(f'{val_dataset_name}@{iteration}:', batch_idx, '/', len(val_data_loader), '|', args.experiment_id)
-				print('REF: {labels} "{ref_}"'.format(ref_ = r['alignment']['ref'] if analyze else r['ref'], **r))
-				print('HYP: {labels} "{hyp_}"'.format(hyp_ = r['alignment']['hyp'] if analyze else r['hyp'], **r))
-				print('WER: {labels} {wer:.02%} | CER: {cer:.02%}\n'.format(**r))
+				print('REF: {labels_name} "{ref_}"'.format(ref_ = r['alignment']['ref'] if analyze else r['ref'], **r))
+				print('HYP: {labels_name} "{hyp_}"'.format(hyp_ = r['alignment']['hyp'] if analyze else r['hyp'], **r))
+				print('WER: {labels_name} {wer:.02%} | CER: {cer:.02%}\n'.format(**r))
```

Comment:
r["labels"] doesn't exist anymore, now it's called r["labels_name"], according to vadimkantorov.